### PR TITLE
2to3: update grammar and examples

### PR DESCRIPTION
**Pull request by bl-ue**
Fixed some glaring mistakes in grammar and added a new example.

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by marchersimon — APPROVED** [review]

File: pages/common/2to3.md [hunk 1]
```diff
@@ -1,28 +1,32 @@
...
-`2to3 {{path/to/file.py}}`
-
 - Convert a Python 2 file to Python 3:
 
 `2to3 --write {{path/to/file.py}}`
 
-- Convert a specified Python 2 feature to Python 3:
+- Display the changes that would be performed without performing them (dry-run):
+
+`2to3 {{path/to/file.py}}`
```
Comment (sbrl) [inline pages/common/2to3.md:12]: I'd suggest this example belongs at the beginning as it was before.

File: pages/common/2to3.md [hunk 2]
```diff
@@ -1,28 +1,32 @@
...
+
+- Convert specific Python 2 features to Python 3:
 
-`2to3 --write {{file.py}} --fix={{raw_input}} --fix={{print}}`
+`2to3 --write {{path/to/file.py}} --fix={{feature1}} --fix={{feature2}}`
```
Comment (sbrl) [inline pages/common/2to3.md:16]: I like the specific examples in the `--fix` here. Let's keep them as-is.

File: pages/common/2to3.md [hunk 3]
```diff
@@ -1,28 +1,32 @@
...
 
-- Convert all features except *raw_input* the specified ones to Python 3:
+- Convert all Python 2 features except the specified ones to Python 3:
 
-`2to3 --nofix={{raw_input}} --fix={{print}} example.py`
+`2to3 --write {{path/to/file.py}} --nofix={{feature1}} --nofix={{feature2}}`
```
Comment (sbrl) [inline pages/common/2to3.md:20]: Same here.

File: pages/common/2to3.md (hunk 1 above)
Comment (bl-ue) [reply pages/common/2to3.md:12]: Ah yes, I agree.  I think I used it because I think it's more common, but it's better to keep it at the top.

**Comment by marchersimon** [comment]
Ping @bl-ue. I guess that got lost in your notifications.

**Review by marchersimon — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @bl-ue!

**Pull request merged**